susetest/python/susetest_api/assertions.py

```diff
@@ -18,6 +18,22 @@ def run_cmd(node, command, msg, time_out = 60):
         node.journal.success("{} OK! ".format(msg))
         return True
 
+# this expetion need to be cached.
+def runOrRaise(node, command, msg, time_out = 60):
+        ''' exec command on node, with msg (fail/succ message for journal)
+            and use tiimeout = 60 as default, but as optional param. so can be modified.'''
+        node.journal.beginTest("{}".format(msg))
+        status = node.run(command, timeout = time_out)
+        if not status and status.code != 0 :
+               node.journal.failure("{} FAIL!".format(msg))
+               raise susetest.SlenkinsError(1)
+        node.journal.success("{} OK! ".format(msg))
+        return True
```

Comment:
RunOrRaise is to break a fondamental test, if fail, everything fail. ( Like init a server , no need for running other tests)